Aztec/Classes/Private/Libxml2/Data/LeafNode.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+/// Represents any node that can be a leaf in the DOM tree.
+///
+protocol LeafNode {
+    
+    /// Returns the text representation of the node.
+    ///
+    func text() -> String
```

Comment:
I see your point on the declaration side but when you are the using the property, it's more clear to just do element.text than element.text(). And has apple say it's better to make it easier on the side of using than on the side of declaring it, because you are probably going to use it a lot more.